Code/MinimalLib/tests/tests.js

```diff
@@ -41,7 +41,7 @@ function test_basics() {
     assert.equal(mol2.get_smiles(),"Oc1ccccc1");
 
     var mjson = mol.get_json();
-    assert(mjson.search("commonchem")>0);
+    assert(mjson.search("rdkitjson")>0);
```

Comment:
Sorry about not catching this when I made the changes to the JSON generation

Reply:
No probs!